### PR TITLE
Start deploying and configuring

### PR DESCRIPTION
**Pull request by jophish**
Make deployments more configurable, add addresses for our initial FarmBot/RevoFees contracts.

### CODE REVIEW

File: deploy/03-configure-farm/index.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  console.log(`Performing ${action}...`);
+  const result = await (await tx).wait();
+  console.log(`${action} done at tx ${result.transactionHash}`);
+};
+
+const main: DeployerFn<{}> = async ({
+  deployer,
+}) => {
+
+  const FARM_ADDRESS = "0xc6686060A1BFa583566Ebca400A2C8771b20Cb8C"
```
Comment (cajubelt) [inline deploy/03-configure-farm/index.ts:19]: we could also get this from the "deployments" directory

File: deploy/03-configure-farm/index.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  if (lpBalance.gt(ethers.BigNumber.from(0))) {
+    // Approve farm to spend it
+    await ERC20__factory.connect(STAKING_TOKEN_ADDRESS, deployer).approve(FARM_ADDRESS, lpBalance)
+    // Deposit LP
+    await (await UbeswapFarmBot__factory.connect(FARM_ADDRESS, deployer).deposit(lpBalance)).wait()
+  }
+
+  // Withdraw it...
+  //await (await UbeswapFarmBot__factory.connect(FARM_ADDRESS, deployer).withdrawAll()).wait()
```
Comment (cajubelt) [inline deploy/03-configure-farm/index.ts:44]: seems like this should probly go in a different step

Comment (cajubelt) [reply deploy/03-configure-farm/index.ts:44]: doesn't look config related

File: deploy/03-configure-farm/index.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+}) => {
+
+  const FARM_ADDRESS = "0xc6686060A1BFa583566Ebca400A2C8771b20Cb8C"
+
+  const COMPOUNDER_ADDRESS = "0x642abB1237009956BB67d0B174337D76F0455EDd"
+  const COMPOUNDER_ROLE = await UbeswapFarmBot__factory.connect(FARM_ADDRESS, deployer).COMPOUNDER_ROLE()
+
+  await doTx(
+    "Setting compounder role",
+    UbeswapFarmBot__factory.connect(FARM_ADDRESS, deployer).grantRole(
```
Comment (cajubelt) [inline deploy/03-configure-farm/index.ts:26]: this file probly needs a similar warning about commenting out before compiling for the first time

File: deploy/02-mcusd-mceur-farm/index.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+const main: DeployerFn<{}> = async ({
+  deployCreate2,
+  deployer,
+}) => {
+  const OWNER_ADDRESS = "0x642abB1237009956BB67d0B174337D76F0455EDd" // Temporary owner, should change to multi-sig
+  const RESERVE_ADDRESS = "0x99649aF776ff1b024F12e8Fe9dfA59A6c0b4bD9C" // Multi-sig owner
+  const STAKING_REWARDS_ADDRESS = "0x2Ca16986bEA18D562D26354b4Ff4C504F14fB01c" // MoolaStakingRewards address
+  const STAKING_TOKEN_ADDRESS = "0xf94fea0c87d2b357dc72b743b45a8cb682b0716e" // mcUSD-mcEUR LP address
+  const REVO_FEES_ADDRESS = "0xa8481Db4C3C7B4F3ce7E28076bD3d7E877f60B62" // RevoFees address
```
Comment (cajubelt) [inline deploy/02-mcusd-mceur-farm/index.ts:12]: we could get this from the deployments directory if wanted

**Review by cajubelt — COMMENTED** [review]
nice job! thanks for doing this!

File: deploy/03-configure-farm/index.ts (hunk 3 above)
Comment (jophish) [reply deploy/03-configure-farm/index.ts:26]: The only file you need to comment out is `/deploy/index.ts` I believe.

Comment (cajubelt) [reply deploy/03-configure-farm/index.ts:25]: oh that's good

**Review by cajubelt — APPROVED** [review]
🥳

**Pull request merged**